src/tests/integration-tests/public-holdays.service.test.ts

```diff
@@ -0,0 +1,56 @@
+import axios from "axios";
+import { checkIfTodayIsPublicHoliday, getListOfPublicHolidays, getNextPublicHolidays } from "../../services/public-holidays.service";
+import { PublicHolidayShort } from "../../types";
+
+describe('Get list of public holidays', () => {
+
+  test('should return an array with proper fields', async () => {
+    const result: PublicHolidayShort[] = await getListOfPublicHolidays(2024, 'FR');
```

Comment:
You use `FR` 3 times in your test, it's better to create a variable for it (e.g. `const testCountryFR = 'FR'`) and place under imports. In this case if you need to change incoming parameter, you will have to fix it only once.